geoips/plugins/modules/readers/modis_hdf4.py

```diff
@@ -44,21 +44,21 @@
 try:
     from pyhdf.HDF import ishdf
 except ImportError:
-    print(
+    LOG.info(
```

Comment:
I'm going to run the full_test.sh to make sure none of these error due to LOG not being defined, etc.

Reply:
I think we can refactor these a little bit to make LOG available and make the error messages more clear. I'll take a look at this before I leave for the day.

Reply:
Created #338